platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -3,26 +3,32 @@
 
 
-## Required accessibility checklist for staging review
+## Required accessibility tests
 
-- [ ] [Axe scans in daily development](#axe-scans-in-daily-development)
-- [ ] [Axe scans in end-to-end tests](#axe-scans-in-end-to-end-tests)
-- [ ] [Color tests](#color-tests)
-- [ ] [Content resize check](#content-resize-check)
-- [ ] [Keyboard navigation check](#keyboard-navigation-check)
-- [ ] [Screen reader tests](#screen-reader-tests)
+- Axe scans in daily development - [read more about the axe browser plugin](#axe-scans-in-daily-development)
+- Axe scans included in end-to-end (e2e) tests - [read more about e2e tests with axe checks](#axe-scans-in-end-to-end-tests)
+- Color tests for proper contrast and colorblindness - [read more about contrast and colorblindness](#color-tests)
+- Content zoomed to 200%, 300%, 400% - [read more about zoomed layouts](#content-resize-check)
+- Keyboard navigation - [read more about navigating with the keyboard](#keyboard-navigation-check)
+- Screen reader tests (VoiceOver + Safari or NVDA + Firefox) - [read more about getting started with screen readers](#screen-reader-tests)
+
+## Recommended accessibility tests
+
+- Keyboard end-to-end tests. These tests are currently written in Nightwatch.js, and will be ported to Cypress.io in the future. [Read more about the legacy Nightwatch tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/testing/508-automated-testing.md)
+- Axe-coconut scans in daily development - [read more about the axe-coconut experimental plugin](https://www.deque.com/blog/test-leading-edge-accessibility-axe-coconut-axe-core-3-0/)
+- WAVE tool spot checks - [read more about the WebAIM WAVE tool](https://wave.webaim.org/)
 
 
 ## How to document that your team has completed the checklist
 
 1. Choose one of these 2 options: 
 
-    * **(preferred)** Use the [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/projects/overview/13). The accessibility test cases template is available in TestRail for you to copy into your test plan and includes all the required accessibility checks.   \
+    * **(preferred)** Use the [TestRail VSP foundational accessibility testing checklist](https://dsvavsp.testrail.io/index.php?/projects/overview/13). The accessibility test cases template is available in TestRail for you to copy into your test plan and includes all the required accessibility checks.   \
 OR
-    *   Copy the checklist from this doc into a doc in your product folder. _Note that this artifact can be used for the Collaboration Cycle staging review._
+    *   Copy the required and recommended checklists from this doc into a doc in your product folder. _Note that this artifact can be used for the Collaboration Cycle staging review._
 
 2. Link to any unresolved accessibility ZenHub issues in your staging review request ticket.
```

Comment:
When they link to any unresolved issues (I don't think we need to say Zenhub), can we provide an example? When they provide a link to their staging a11y review feedback instead of the linked list of individual issues I have been creating the list myself, so that I have it for the 508 Office.

Reply:
For the section below on how to request help, I don't understand the first bullet. I think it says if they have specific questions about issues from accessibility checks to open a 508 issue. Instead, I think we want to say, if you find issues during your team's own accessibility tests, open a 508 issue and then if not resolved… the next bullet.